src/hdmint/tracker.h

```diff
@@ -52,7 +53,7 @@ class CHDMintTracker
     bool UnArchive(const uint256& hashPubcoin, bool isDeterministic);
     bool UpdateZerocoinEntry(const CSigmaEntry& zerocoin);
     bool UpdateState(const CMintMeta& meta);
-    void SetMetaNonDeterministic();
+    std::set<uint256> GetMempoolTxids();
```

Comment:
does not make sense to define here as public function. I think you should move to `src/main.cpp` as static function or set it as private to allow only internal use.